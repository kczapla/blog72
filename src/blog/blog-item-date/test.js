import React from 'react'
import renderer from 'react-test-renderer'

import BlogItemDate from './index'

it('renders correctly', () => {
  const tree = renderer
    .create(<BlogItemDate date={'2020-01-01'}/>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
