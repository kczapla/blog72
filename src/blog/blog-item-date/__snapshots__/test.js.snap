// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<time
  dateTime={"2020-01-01T00:00:00.000Z"}
>
  Jan 01, 2020
</time>
`;
